pkg/shell/machine-dialogs.js

```diff
@@ -720,6 +722,9 @@ define([
             error_options = ex;
             keys = credentials.keys_instance();
             $(keys).on("changed", update_keys);
+            cockpit.user().done(function (user) {
+                self.user = user;
+            });
```

Comment:
Should we wait to call render() until the above promise completes? ie: .always()?